xla/tests/hlo_test_base.h

```diff
@@ -432,6 +432,12 @@ class HloTestBase : public ManifestCheckingTest {
   static se::Platform* GetReferencePlatform();
   static se::Platform* GetTestPlatform();
 
+  // Compares the inputs shapes of two modules and returns the list of parameter
```

Comment:
Can you please split it into a separate change.

Reply:
@shraiysh it seems this comment is not addressed yet? It is usually better to do such refactorings separately as they are no functional change, so less likely that they need to be rolled back, and it makes PRs smaller and easier to review.

Reply:
Added #14822